### PR TITLE
WRQ-18295: Reactivate skipped ui-tests.

### PR DESCRIPTION
**Pull request by dongsuwon**
Enact-DCO-1.0-Signed-off-by: Dongsu Won (dongsu.won@lgepartner.com)

### Checklist
* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
- Need to reactivate skipped ui-tests

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
- Reactivate skipped ui-tests

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]:# (Related issues, references)
WRQ-18295

### CODE REVIEW

File: tests/ui/specs/VirtualList/VirtualList/onKeyDown/VirtualList-QWTC-2062-specs.js [hunk 1]
```diff
@@ -10,51 +10,50 @@ describe('onKeyDown [QWTC-2062]', function () {
 		await Page.spotlightSelect();
 		await Page.buttonLeft.moveTo();
 		await Page.spotlightRight();
-		await expectFocusedItem(0, 'focus 1');
+		await expectFocusedItem(0);
```
Comment (juwonjeong) [inline tests/ui/specs/VirtualList/VirtualList/onKeyDown/VirtualList-QWTC-2062-specs.js:13]: Could you please explain why we need this change?

File: tests/ui/specs/SwitchItem/SwitchItem-specs.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 const Page = require('./SwitchItemPage');
 
 // Skipping until SwitchItem is updated to handle the new Item structure
```
Comment (juwonjeong) [inline tests/ui/specs/SwitchItem/SwitchItem-specs.js:3]: I think we need to remove this comment because we activate it.

Comment (dongsuwon) [reply tests/ui/specs/SwitchItem/SwitchItem-specs.js]: You are correct. I removed this comment.

**Review by juwonjeong — APPROVED** [review]
LGTM

**Pull request merged**